--- a/sounddevice.py
+++ b/sounddevice.py
@@ -78,11 +78,14 @@
             _libname = 'libportaudio' + _platform.architecture()[0] + '-asio.dll'
         else:
             _libname = 'libportaudio' + _platform.architecture()[0] + '.dll'
+    elif _platform.system() == 'Linux':
+        _libname = 'libportaudio.so.2'
     else:
         raise
-    import _sounddevice_data
-    _libname = _os.path.join(
-        next(iter(_sounddevice_data.__path__)), 'portaudio-binaries', _libname)
+    if _platform.system() in ('Darwin', 'Windows'):
+        import _sounddevice_data
+        _libname = _os.path.join(
+            next(iter(_sounddevice_data.__path__)), 'portaudio-binaries', _libname)
     _lib = _ffi.dlopen(_libname)
 
 _sampleformats = {
